src/app/start-page/start-page.scss

```diff
@@ -0,0 +1,110 @@
+$salmon: rgb(255, 82, 82);
+$brown: rgb(116, 32, 6);
+
+html {
+  height: 100%;
+}
+
+.img {
+  background-image: url("../../resources/start-page.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100%;
+}
+
+body {
+  height: 100%;
+}
+
+.md-button {
+  &.md-fab {
+    background-color: $salmon;
+    &:not([disabled]):hover {
+      background-color: rgb(197, 65, 65);
+    }
+  }
+}
+
+h1,
+.reg {
+  padding-top: 10%;
+  text-shadow: 1px 1px 2px black;
+}
+
+.salmon {
+  color: $salmon;
+}
+
+.skyblue {
+  color: skyblue;
+}
+
+.brown {
+  color: $brown;
+}
+
+.container {
+  height: 35px;
+  width: 350px;
+}
+
+md-input-container:not(.md-input-invalid) .md-input {
+  color: $salmon;
+}
+md-input-container:not(.md-input-invalid).md-input-focused .md-input {
+  border-color: $salmon;
+}
+md-input-container:not(.md-input-invalid) .md-input {
+  border-color: rgb(172, 172, 172);
+}
+md-input-container:not(.md-input-invalid).md-input-focused label {
+  color: $salmon;
```

Comment:
You can join this style with the '.salmon' and the 'md-input-container:not(.md-input-invalid) .md-input'